selfservice/hook/web_hook_integration_test.go

```diff
@@ -1015,7 +1046,7 @@ func TestAsyncWebhook(t *testing.T) {
 	err := wh.ExecuteLoginPostHook(nil, req, node.DefaultGroup, f, s)
 	require.NoError(t, err) // execution returns immediately for async webhook
 	select {
-	case <-time.After(200 * time.Millisecond):
+	case <-time.After(200 * time.Minute):
```

Comment:
I guess revert this? The test will time-out before we wait for over 3h :sweat_smile: 
In case you hit this timeout, I would rather increase this to maybe 500ms.

Reply:
Uh, good catch. This shouldn't have been committed